card/mod/05_email/set/all/notify.rb

```diff
@@ -68,7 +68,7 @@ def silent_change?
 end
 
 def notable_change?
-  !silent_change? && !supercard && current_act &&
+  !silent_change? && !supercard && current_act && current_act.card_id == id &&
```

Comment:
I think Philipp is right. I thought the `!supercard` will block the sending but supercard was `nil` for the changed card that is part of the act( I am not sure because it was almost a month ago). I tried to looking to params I can use and then I though comparing the current act id and current card might be ok to stop that as there is a way to include all the changes in the email.